index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Register</title>
+  </head>
+  <link href="./CSS/style.css" rel="stylesheet" />
+  <script>
+
+  </script>
+  <body>
+    <div class="header"><header>Assignment 1</header></div>
+    <h1>Register</h1>
+    <form>
+      <div class="values">
+        <div class="fname">
+          <label for="first"> First Name:</label>
+          <input type="text" name="first" id="first" class="input" />
+        </div>
+       <div class="lname">
+         <label for="last"> Last Name:</label
+         ><input type="text" name="last" id="last" class="input" />
+         </div>
+        
+        <div class="email"><label for="email">Email:</label>
+          <input type="email" name="email" id="email" class="input" />
+        </div>
+        <div class="gender"><label for="gender">Gender:</label>
+          <input type="radio" id="male" name="gender" valu="male" />
```

Comment:
refer https://linuxhint.com/get-value-of-selected-radio-button-using-javascript/
to get radio value